tests/UI/Component/Dropzone/File/DropzoneRendererTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+require_once(__DIR__ . "/../../../../../libs/composer/vendor/autoload.php");
+require_once(__DIR__ . "/../../../Base.php");
+
+/**
+ * Class FileDropzoneRendererTest
+ *
+ * @author  nmaerchy
+ * @date    08.05.17
+ * @version 0.0.1
+ *
+ */
+class DropzoneRendererTest extends ILIAS_UI_TestBase {
```

Comment:
I added some more, but unfortunately at the moment we can't test the JS-stuff...